### ISSUE TITLE
Fix attention mask warning

### ISSUE THREAD

**Issue by Pchatain** [scottviteri/MarkovianTraining#13]
Check out why it's warning me that the attention mask isn't set and verify this is due to batching.
```
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.
```
